php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml

```diff
@@ -0,0 +1,14 @@
+translation:
+    locales: ["en", "sv", "fr"]
+    webui:
+        enabled: true
+    edit_in_place:
+        enabled: false
+        config_name: app
+        activator: php_translation.edit_in_place.activator
+    configs:
+        app:
+            dirs: ["%kernel.project_dir%/templates", "%kernel.root_dir%/src"]
+            output_dir: "%kernel.project_dir%/app/Resources/translations"
+            excluded_names: ["*TestCase.php", "*Test.php"]
+            excluded_dirs: [cache, data, logs]
```

Comment:
What does it refer to? `var/`? Why?

Reply:
These values will be given to `Finder::notPath()`. Ie, when extracting the `src` or `vendor` for translations we should never go in to folders named "cache", "data" or "logs".